jaxrs/resteasy-client/src/test/java/org/jboss/resteasy/test/internal/ClientInvocationBuilderTest.java

```diff
@@ -0,0 +1,138 @@
+package org.jboss.resteasy.test.internal;
+
+import java.io.ByteArrayOutputStream;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.client.AsyncInvoker;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.Invocation.Builder;
+import javax.ws.rs.core.MediaType;
+
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.client.jaxrs.ResteasyWebTarget;
+import org.jboss.resteasy.client.jaxrs.internal.ClientInvocation;
+import org.jboss.resteasy.core.Dispatcher;
+import org.jboss.resteasy.spi.ResteasyDeployment;
+import org.jboss.resteasy.test.EmbeddedContainer;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ClientInvocationBuilderTest {
```

Comment:
I don't see ClientInvocationBuilderTest ... . I would think that it's somewhat different, though, since Nicolas has changed the behavior of ClientInvocationBuilder. But this test should be moved from resteasy-client to be consistent with our current testsuite structure. The easiest thing might be to remove the call to invoke and make it a pure unit test in resteasy-unit-tests.